docs/framework/configure-apps/file-schema/runtime/loadfromremotesources-element.md

```diff
@@ -89,26 +89,26 @@ Setting the `<loadFromRemoteSources>` element to `true` prevents this exception
 
 ## Notes
 
-- In the .NET Framework 4.5 and later versions, assemblies on local network shares run in full trust by default; you do not have to enable the `<loadFromRemoteSources>` element.
+- In the .NET Framework 4.5 and later versions, assemblies on local network shares (that is, [Local Intranet Security zone](/deployedge/per-site-configuration-by-policy#windows-security-zones)) run in full trust by default; you do not have to enable the `<loadFromRemoteSources>` element. For Security zones other than Local Machine or Local Intranet, the value must be set to `true`.
```

Comment:
```suggestion
- In .NET Framework 4.5 and later versions, assemblies on local network shares (that is, the [Local Intranet security zone](/deployedge/per-site-configuration-by-policy#windows-security-zones)) run in full trust by default; you don't have to enable the `<loadFromRemoteSources>` element. For security zones other than Local Machine or Local Intranet, set the value to `true`.
```

Reply:
This page is in archived content, but might be a better place to point to?

https://learn.microsoft.com/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ms537183(v=vs.85)#default-url-security-zones